scripts/extract_planning.R

```diff
@@ -86,24 +88,27 @@ planning <- left_join(
         image_view_url),
 
     by = c('aggregated_crossings_id' = 'stream_crossing_id')) %>%
+
+    # filtering where pscis_status is NA or not equal to 'HABITAT CONFIRMATION' and barrier_status is not equal to
+    # 'PASSABLE' or 'UNKNOWN'
     filter(is.na(pscis_status) | (pscis_status != 'HABITAT CONFIRMATION' &
                                     barrier_status != 'PASSABLE' &
                                     barrier_status != 'UNKNOWN')) %>%
     ### since we are deep in salmon country here lets use a coho metric.  Lets change this to look at everything with
     ### over 1km of rearing habitat to start. Don't forget about fpr_dbq_lscols .  Also - if not familiar have a look at
     ###  our tables in methods of past reports (Skeena has salmon) which explain the thresholds in general. Look at the
     ### csv in bcfishpass that decided what they are too though because they are new!
-    filter(bt_rearing_km > 0.3) %>%
+    filter(bt_rearing_km > 1) %>%
     filter(crossing_type_code != 'OBS') %>%
     filter(is.na(barriers_anthropogenic_dnstr)) %>%
-    ### this will not work because we have no pdf maps for this area.
-    ### Can be useful (mostly for other collaborators) when we do. can nuke
-    mutate(map_link = paste0('https://hillcrestgeo.ca/outgoing/fishpassage/projects/parsnip/archive/2022-05-27/FishPassage_', dbm_mof_50k_grid, '.pdf')) %>%
```

Comment:
just a heads up the these links will probably be back eventually and that we now are organizing our maps on the hillcrest server according to watershed group code (ex. pars vs parsnip) as that will allow us to construct these links using bcfishpass.crossings `wateshed_group_code` variable as well as the`dbm_mof_50k_grid` variable .

The archive dates are there because we update the maps after fieldwork so we wanted a snapshot in time. 

just an FYI